examples/bw_tester/bw_tester.c

```diff
@@ -101,18 +119,11 @@ int main(int argc, char **argv)
     goto close;
   }
 
-  ret = fla_object_create(fs, pool, &obj);
```

Comment:
I think to start lets just wrap the following code in a loop. I think I wasn't so explicit earlier, the individual object read and write BW is still useful for the bw_tester. I have been testing large objects so each object gives meaningful performance numbers. I am assuming that mkfs.flexalloc would be run between runs of bw_tester so we don't have to worry about cleanup.

Reply:
Wrapping the `fla_object_create` in a loop is what I intended to do. If you look at line 39 of the new file that line is inside the loop that goes through all the objects that need to be created. Are you hitting some issues?